GoogleDataLogger/GoogleDataLogger/Classes/GDLLogger.m

```diff
@@ -31,18 +33,34 @@ @interface GDLLogger ()
 
 @implementation GDLLogger
 
-- (instancetype)initWithLogSource:(NSInteger)logSource
+- (instancetype)initWithLogMapID:(NSString *)logMapID
                   logTransformers:(nullable NSArray<id<GDLLogTransformer>> *)logTransformers
                         logTarget:(NSInteger)logTarget {
   self = [super init];
   if (self) {
-    NSAssert(logSource > 0, @"A log source cannot be negative or 0");
+    NSAssert(logMapID.length > 0, @"A log source cannot be negative or 0");
```

Comment:
Nit: the error message could be a bit better here, `A log source cannot be empty or nil.`